src/SurveyService.Business/Commands/Question/GetQuestionCommand.cs

```diff
@@ -0,0 +1,84 @@
+using LT.DigitalOffice.Kernel.Helpers.Interfaces;
+using LT.DigitalOffice.Kernel.Responses;
+using LT.DigitalOffice.Models.Broker.Models;
+using LT.DigitalOffice.SurveyService.Broker.Requests.Interfaces;
+using LT.DigitalOffice.SurveyService.Business.Commands.Question.interfaces;
+using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.SurveyService.Mappers.Responses.Interfaces;
+using LT.DigitalOffice.SurveyService.Models.Db;
+using LT.DigitalOffice.SurveyService.Models.Dto.Models;
+using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question.Filters;
+using LT.DigitalOffice.SurveyService.Models.Dto.Responses.Question;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.SurveyService.Business.Commands.Question;
+
+public class GetQuestionCommand : IGetQuestionCommand
+{
+  private readonly IQuestionRepository _repository;
+  private readonly IQuestionResponseMapper _questionResponseMapper;
+  private readonly IResponseCreator _responseCreator;
+  private readonly IUserService _userService;
+  private readonly IOptionInfoMapper _optionInfoMapper;
+  private readonly IUserInfoMapper _userInfoMapper;
```

Comment:
Do you need it here?

Reply:
Why not?

Reply:
You don't use it

Reply:
IUserService - line 95
IOptionINfoMapper - line 101
IUserInfoMapper - line 97

Reply:
If you are about IserInfoMapper - fixed